sdk/cognitiveservices/ms-azure-cs-inkrecognizer/README.md

```diff
@@ -0,0 +1,131 @@
+
+## Getting started
+
+### Prerequisites
+
+Android 6.0 or later
+Java Development Kit (JDK) with version 7 or above
+You must have a [Cognitive Services API account][cog_serv_acc]. If you don't have an Azure subscription, you can [create an account][create_acc] for free. You can get your subscription key from the [Azure portal][az_portal] after creating your account, or [Azure website][az_web] after activating a free trial.
+
+## Key concepts
+
+### Implement InkStroke and InkPoint
+
+The InkStroke interface represents an ink stroke (a collection of ink points from the time a user places the writing instrument on the writing surface until the the instrument is lifted. You will be expected to implement this interface so that the InkRecognizer Client object can use it to translate the ink to JSON for delivery to the Ink Recognizer service.
+
+```Java
+public interface InkStroke {
+     Iterable<InkPoint> getInkPoints();
+     InkStrokeKind getKind();
+     long getId();
+     String getLanguage();
+} 
+```
+
+The InkPoint interface represents a single position on the path of an ink stroke. You are expected to implement this interface as well.
+
+```Java
+public interface InkPoint {
+    float getX();
+    float getY();
+}
+```
+
+The StrokeKind enum represents the class a stroke belongs to. You are expected to set this value when it is known with absolute certainty. The default value is "UNKNOWN".
+
+```Java
+public enum InkStrokeKind {
+    UNKNOWN("unknown"),
+    DRAWING("inkDrawing"),
+    WRITING("inkWriting");
+}
+```
+
+### Create client
+
+You will need to then create an InkRecognizerClient object as follows:
+
+```Java
+InkRecognizerClientBuilder inkRecognizerClientBuilder = new InkRecognizerClientBuilder();
+InkRecognizerClient inkRecognizerClient = inkRecognizerClientBuilder.credentials(credential)
+                .endpoint(endpoint)
+                .displayMetrics(displayMetrics)
+                .buildClient();
+```
+
+or an InkRecognizerAsyncClient object as shown as:
+
+```Java
+InkRecognizerClientBuilder inkRecognizerClientBuilder = new InkRecognizerClientBuilder();
+InkRecognizerAsyncClient inkRecognizerAsyncClient = inkRecognizerClientBuilder.credentials(credential)
+                .endpoint(endpoint)
+                .displayMetrics(displayMetrics)
+                .buildAsyncClient();
+```
+
+### Send request
+
+You can then send the strokes for processing to the service through the client
+
+```Java
+InkRecognitionRoot root = inkRecognizerClient.recognizeInk(strokes);
+```
+
+### Use the response
+
+You can call methods on the InkRecognitionRoot. For example, you can search for specific recognized words in the recognized text as follows:
+
+```Java
+Iterable<InkWord> wordsFound = root.findWord("hello")
+```
+
+## Examples
+
+The [Ink Recognizer Samples][samples] cover some of the common use cases of the Ink Recognizer SDK.
+
+## Troubleshooting
+
+## Next steps
+
```

Comment:
similarly could you fill in or remove this section?

Reply:
I do not have anything concrete for these sections. But removing them seems to cause the README validation check to fail. I will try to find something to fill these sections.

Reply:
Filled.